app/experimenter/visualization/api/v3/views.py

```diff
@@ -79,7 +81,7 @@ def append_population_percentages(data):
         )
 
 
-def computeSignificance(lower, upper):
```

Comment:
XD this ain't js!  I'm surprised linting didn't complain about this....

Reply:
I know right! Hopping between python and js has me going like 
![image](https://user-images.githubusercontent.com/784781/97620111-770eb080-19f7-11eb-858d-a479221137dc.png)

Reply:
Welcome to my life XD

Reply:
this emoji is so extra